.github/workflows/new_good_first_issue.yml

```diff
@@ -0,0 +1,24 @@
+name: Discord Notification - New Issue
+
+on:
+  issues:
+    types: [opened]
```

Comment:
What if someone opened an issue, and **then** I tagged it as a Good First Issue?

Reply:
There won't be a response right now. It wasn't mentioned in the issue as a preferred result.
If you'd like, I can add this functionality to the code.

Reply:
The most recent commit includes a fix.